src/core/interfaces/ISablierLockupBase.sol

```diff
@@ -314,11 +322,26 @@ interface ISablierLockupBase is
     /// - `to` must not be the zero address.
     /// - `amount` must be greater than zero and must not exceed the withdrawable amount.
     /// - `to` must be the recipient if `msg.sender` is not the stream's recipient or an approved third party.
+    /// - The `msg.value` must not be less than the Sablier fee.
```

Comment:
This can be removed since there is no concept of Sablier fee yet.

Reply:
https://github.com/sablier-labs/v2-core/pull/1087/commits/1f8d30b76893c80e77a6c6a1ea24737fa32b81ba